Tests/SafeDIToolTests/SafeDIToolTests.swift

```diff
@@ -2732,13 +2732,11 @@ final class SafeDIToolTests: XCTestCase {
         line: UInt = #line,
         block: () async throws -> ReturnType
     ) async {
-        var didThrow = false
         do {
             _ = try await block()
+            XCTFail("Did not throw error!", line: line)
         } catch {
-            didThrow = true
-            XCTAssertEqual((error as CustomStringConvertible).description, errorDescription, line: line)
+            XCTAssertEqual("\(error)", errorDescription, line: line)
```

Comment:
This change was required to get tests working on Linux. `(error as CustomStringConvertible).description` was not finding our `var description: String` code on Linux for unknown reasons.